src/main/java/com/solvd/laba/carina/saucelabs/utils/NavigationUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.solvd.laba.carina.saucelabs.utils;
+
+import com.solvd.laba.carina.saucelabs.android.CartScreen;
+import com.solvd.laba.carina.saucelabs.android.ProductDetailsScreen;
+import com.solvd.laba.carina.saucelabs.common.LoginScreenBase;
+import com.solvd.laba.carina.saucelabs.android.ProductsScreen;
+import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.utils.R;
+
+public class NavigationUtils implements IAbstractTest {
```

Comment:
```suggestion
public class MobileNavigationUtils implements IAbstractTest {
```
since you already have NavigationUtils for web I think better to rename this class to avoid confusion

Reply:
Done.